cross/arm/tryrun.cmake

```diff
@@ -1,6 +1,7 @@
 SET( HAVE_SHM_OPEN_THAT_WORKS_WELL_ENOUGH_WITH_MMAP_EXITCODE
      0
      CACHE STRING "Result from TRY_RUN" FORCE)
+
```

Comment:
Yeah my bad sorry, saw a + and assumed it was a literal + forgetting it's used to mean addition.